### PR TITLE
BAEL-6035 muasif80@gmail.com

### PR DESCRIPTION
**Pull request by muasif80**

### CODE REVIEW

File: spring-security-modules/spring-security-saml-2/pom.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>spring-security-saml-2</artifactId>
+	<version>1.0-SNAPSHOT</version>
+	<name>spring-security-saml-2</name>
+
```
[the remark this answers is not in the document]
Comment (KevinGilmore) [reply spring-security-modules/spring-security-saml-2/pom.xml:8]: Finally, you'll need to edit the parent (spring-security-modules) pom.xml file's "modules" section to ensure your new module is part of the build process.

File: spring-security-modules/spring-security-saml-2/src/main/resources/templates/index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="https://www.thymeleaf.org"
+	xmlns:sec="https://www.thymeleaf.org/thymeleaf-extras-springsecurity5">
+
+<head>
```
Comment (KevinGilmore) [inline spring-security-modules/spring-security-saml-2/src/main/resources/templates/index.html:5]: Let's also use four-space (no tabs) indentation here.

**Review by KevinGilmore — COMMENTED** [review]
Please correct the indentation in several files, plus a lot of other things noted in the pom.xml file.

File: spring-security-modules/spring-security-saml-2/pom.xml [hunk 3]
```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
```
[the remark this answers is not in the document]
Comment (muasif80) [reply spring-security-modules/spring-security-saml-2/pom.xml]: Done

File: spring-security-modules/spring-security-saml-2/.gitignore [hunk 4]
```diff
@@ -0,0 +1,33 @@
+HELP.md
```
[the remark this answers is not in the document]
Comment (muasif80) [reply spring-security-modules/spring-security-saml-2/.gitignore]: Done

File: spring-security-modules/spring-security-saml-2/pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (muasif80) [reply spring-security-modules/spring-security-saml-2/pom.xml:4]: 1. Done - The correction to indentation to use 4 spaces
2. Done - The rearrangement of the sections in pom.xml, put the repositories just before parent section as did not find instruction about it
3. Done - Added the module to parent pom

I added the formatter for eclipse to my STS IDE. I am not sure why it does not format the xml or html files.

File: spring-security-modules/spring-security-saml-2/src/main/resources/templates/index.html (hunk 2 above)
Comment (muasif80) [reply spring-security-modules/spring-security-saml-2/src/main/resources/templates/index.html:5]: Done

File: spring-security-modules/spring-security-saml-2/pom.xml [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+	<artifactId>spring-security-saml-2</artifactId>
+	<version>1.0-SNAPSHOT</version>
+	<name>spring-security-saml-2</name>
```
[the remark this answers is not in the document]
Comment (muasif80) [reply spring-security-modules/spring-security-saml-2/pom.xml]: Done

**Comment by davidmartinezbarua** [comment]
retest this please

**Pull request closed without merging**